Owncloud iOs Client/Files/Edit/EditFileViewController.h

```diff
@@ -19,14 +19,22 @@
 
 #define shareMainViewNibName @"EditFileViewController"
 
+extern NSString *iPhoneDoneEditFileTextMessageNotification;
```

Comment:
Let's move this to the notification utils file an same of the rest of notifications